keyboards/stront/keymaps/zzeneg/config.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 zzeneg (@zzeneg)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+// #define DEBUG_MATRIX_SCAN_RATE
+
+#define I2C_DRIVER I2CD1
+#define I2C1_SDA_PIN GP10
+#define I2C1_SCL_PIN GP11
+
+/* HID is supported only on master, so left must be master */
+#define MASTER_LEFT
+#undef EE_HANDS
+#undef SPLIT_LED_STATE_ENABLE
+#undef SPLIT_MODS_ENABLE
+
+#define CIRQUE_PINNACLE_TAP_ENABLE
+#define POINTING_DEVICE_GESTURES_SCROLL_ENABLE
+#define POINTING_DEVICE_GESTURES_CURSOR_GLIDE_ENABLE
+#define MOUSE_EXTENDED_REPORT
+
+#define TAPPING_TERM 150
+#define TAPPING_TERM_PER_KEY
+#define QUICK_TAP_TERM 0
+#define IGNORE_MOD_TAP_INTERRUPT
```

Comment:
Is default behavior "now"
```suggestion
```

Reply:
I didn't realize it merges today :) Updated & removed.